endpoints/main.go

```diff
@@ -136,20 +151,29 @@ func main() {
 		return
 	}
 	// Data in resources/EndpointSources was taken from https://fhirfetcher.github.io/data.json
-	var listOfEndpoints = fetcher.GetListOfEndpoints(endpointsFile)
+	var listOfEndpoints, err = fetcher.GetListOfEndpoints(endpointsFile)
+	if err != nil {
+		log.Error("Endpoint List Parsing Error: ", err.Error())
```

Comment:
Do any of these errors require `fatal` handling? Like, should we quit the app if we can't parse the endpoints? (this can be a new task)

Reply:
The only fatal error right now is if you don't provide an endpoints list. I do like the idea making this specific error quit out.

Reply:
sounds good! please change to `log.fatal`